google-cloud-logging/lib/google/cloud/logging/v2/config_service_v2_client.rb

```diff
@@ -334,7 +334,7 @@ def list_sinks \
           #       "billingAccounts/[BILLING_ACCOUNT_ID]/sinks/[SINK_ID]"
           #       "folders/[FOLDER_ID]/sinks/[SINK_ID]"
           #
-          #   Example: +"projects/my-project/sinks/my-sink-id"+.
+          #   Example: +"projects/my-project-id/sinks/my-sink-id"+.
```

Comment:
It's unfortunate that in the manual layer, @blowmage's PR #1824 just changed all of the examples that were previously using `my-project-id` to now use `my-project`. 

```ruby
         #   bigquery = Google::Cloud::Bigquery.new(
 -        #     project: "my-project-id",
 -        #     keyfile: "/path/to/keyfile.json"
 +        #     project_id: "my-project",
 +        #     credentials: "/path/to/keyfile.json"
          #   )
          #
 -        #   bigquery.project #=> "my-project-id"
 +        #   bigquery.project_id #=> "my-project"
          #
```

Reply:
I hope we didn't accidentally change the generated GAPIC code. I thought we only changed the manual layer.

Reply:
Yes, @blowmage, in fact PR #1824 did update this file. I was just commenting on the inconsistency as unfortunate, but in fact this is a reversion to the `my-project-id` usage.

Reply:
Oops. Sorry.

Agreed, the inconsistency is unfortunate. We have several code examples where saving the extra 3 characters is worth it though.

Reply:
This comes down directly from [the proto](https://github.com/googleapis/googleapis/blob/70402cea3d4f76878ebe83907c4ccc1c604b39b7/google/logging/v2/logging.proto#L87), so I think we have to live with the inconsistency if the 3 character savings are needed in the high level examples.